docs/pages/style-spec.js

```diff
@@ -308,43 +308,55 @@ class Item extends React.Component {
-                    </em>
+                        <span> <a href={`#types-${this.props.type}`}>{this.props.type}</a></span>}
+                        {'minimum' in this.props && 'maximum' in this.props &&
+                        <span> between <var>{this.props.minimum}</var> and <var>{this.props.maximum}</var> inclusive</span>}
+                        {'minimum' in this.props && !('maximum' in this.props) &&
+                        <span> greater than or equal to <var>{this.props.minimum}</var></span>}
+                        {!('minimum' in this.props) && 'maximum' in this.props &&
+                        <span> less than or equal to <var>{this.props.minimum}</var></span>}. </em>
 
                     {this.props.values && !Array.isArray(this.props.values) && // skips $root.version
-                    <span className='space-right quiet'>
-                            <em>One of</em> {Object.keys(this.props.values)
+                    <em className='quiet'>
+                        One of {Object.keys(this.props.values)
                             .map((opt, i) => <var key={i}>{opt}</var>)
-                            .reduce((prev, curr) => [prev, ', ', curr])}. </span>}
+                            .reduce((prev, curr) => [prev, ', ', curr])}. </em>}
 
                     {this.props.units &&
-                    <span className='space-right quiet'>
-                            <em>Units in</em> <var>{this.props.units}</var>. </span>}
+                    <em className='quiet'>
+                        Units in <var>{this.props.units}</var>. </em>}
 
                     {this.props.default !== undefined &&
-                    <span className='space-right quiet'>
-                            <em>Defaults to </em> <var>{typeof this.props.default !== 'string' ?
-                        JSON.stringify(this.props.default) :
-                        this.props.default}</var>. </span>}
+                    <em className='quiet'>
+                        Defaults to <var>{typeof this.props.default !== 'string' ? JSON.stringify(this.props.default) : this.props.default}</var>. </em>}
 
                     {this.props.requires &&
-                    <span className='space-right quiet'>
-                            {this.props.requires.map((r, i) => this.requires(r, i))} </span>}
+                    <em className='quiet'>
+                        {this.props.requires.map((r, i) => this.requires(r, i))} </em>}
+
+                    {this.props.function === "interpolated" &&
+                    <em className='quiet'>
+                        Supports all <a href='#expressions-curve'><span className='icon smooth-ramp inline'/>curve</a> types. </em>}
+
+                    {this.props.function === "piecewise-constant" &&
+                    <em className='quiet'>
+                        Supports <a href='#expressions-curve'><span className='icon step-ramp inline' /><code>step</code> curves</a> only. </em>}
```

Comment:
🤔  aside from `line-dasharray`, every other style property marked `"piecewise-constant"` is of a non-interpolatable type that's explicitly disallowed in interpolated curves (or `"interpolate"`, after #5528).  Is it still worthwhile to have these notes attached to each style property?  I think probably so, but I'm not totally sure.

Reply:
Yeah, I think it's a useful reminder.